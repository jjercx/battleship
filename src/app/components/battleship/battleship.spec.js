import React from "react";
import Battleship from "./battleship";
import renderer from "react-test-renderer";
import { MemoryRouter } from "react-router-dom";

describe("<Battleship/>", () => {
  it("matches snapshot", () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <Battleship />
        </MemoryRouter>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
