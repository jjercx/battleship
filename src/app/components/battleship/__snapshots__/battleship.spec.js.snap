// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Battleship/> matches snapshot 1`] = `
<div
  className="sc-fznMAR eanBUi"
>
  <div
    className="sc-fznWOq BKMx"
  >
    <div
      className="sc-fznxKY eBcwVQ title"
    >
      BATTLESHIP
    </div>
    <div
      className="sc-fznKkj hxmZlM"
      onClick={[Function]}
    >
      NEW GAME
    </div>
    <div
      className="sc-fznKkj hxmZlM"
    >
      LEADERBOARD
    </div>
    <div
      className="sc-fznKkj hxmZlM"
      onClick={[Function]}
    >
      OPTIONS
    </div>
  </div>
</div>
`;
